docs/deploying.md

```diff
@@ -41,29 +41,46 @@ If you make a change to your prototype, commit your changes as usual then run:
 
 to push your changes to Heroku
 
-## Deploying different versions
+## Deploying different branches
+
+By default when you push to Heroku your master branch will be deployed. Sometimes you may want to deploy code that's on a different branch, eg a test feature.
+
+To deploy a different branch:
+
+`git push heroku branch_name:master`
+
+This pushes your local branch `branch_name` on to the heroku master branch. If Heroku complains that the remote contains work you don't have, you can force push using `-f`:
+
+`git push -f heroku branch_name:master`
+
+
+## Deploying to more than one Heroku app
 
 We have found its helpful to have at least two different versions of a prototype online.
```

Comment:
I don't follow what you're suggesting. I'm not sure it's correct to say that each is typically pushed from a different local branch. Or that people have a 'dev' branch. You also don't need to be on a specific branch to push it to heroku.

This is guidance for what you do if you want the code in a branch (whatever the name) to be on heroku.